iam_policy/templates/jenkins_fargate_integration.json.tpl

```diff
@@ -0,0 +1,78 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Action": [
+        "s3:PutObject",
+        "s3:ListBucket",
+        "s3:GetObject",
+        "iam:PassRole",
+        "ecs:StopTask",
+        "ecs:RunTask",
+        "ecs:ListContainerInstances",
+        "ecs:DescribeTasks"
+      ],
+      "Resource": [
```

Comment:
If this policy is integration specific then should remove the non-integration resources?

Reply:
It will once I've got the prod one up and running but I don't want to remove permissions from this one until there's an alternative way of doing it. That'll come on another PR.